File: python/testes.js
const axios = require('axios');
(async () => {    
    const api_key = 'ff118324a9784dd:bsbyb0uhw8yhf11'
    const response = await axios.get(`https://api.tradingeconomics.com/country/mexico?c=${api_key}`)
    console.log(response.data)
})()

File: python/testes.py
# import tradingeconomics as te
# te.login('ff118324a9784dd:uf05cp73mzkte9u')
# x = te.getMarketsSearch(country='mexico')
# print (x)

import tradingeconomics as te
te.login('ff118324a9784dd:uf05cp73mzkte9u')
x = te.getCmtCountryFilterByType(country1 ='mexico', country2 ='sweden',
 type='export')
print (x)

# te.login('key:ff118324a9784dd:bsbyb0uhw8yhf11')
# x = te.getIndicatorData(country=['mexico', 'sweden'])